src/distributions/mod.rs

```diff
@@ -256,6 +285,30 @@ where
 {
 }
 
+/// A distribution of values of type `S` derived from the distribution `D`
+/// by mapping its output of type `T` through the closure `F`.
+///
+/// This `struct` is created by the [`map`] method on [`Distribution`].
```

Comment:
Just link [`Distribution::map`]? Seems clearer to me.

Reply:
Agreed. I have admittedly copied this from `DistrIter`.